code/modules/mining/fulton.dm

```diff
@@ -10,13 +10,8 @@ var/list/total_extraction_beacons = list()
 	var/list/beacon_networks = list("station")
 	var/uses_left = 3
 	var/can_use_indoors
-	var/safe_for_living_creatures = 0
+	var/safe_for_living_creatures = 1
```

Comment:
Should probably change the description and name so the players know that they can use this on living things, too.